Source/RapyutaSimulationPlugins/Private/Core/RRNetworkGameState.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
+
+// UE
+#include "Core/RRNetworkGameState.h"
+
+// RapyutaSimulationPlugins
+#include "Core/RRNetworkPlayerController.h"
+
+ARRNetworkGameState::ARRNetworkGameState()
+{
+    bReplicates = true;
+    PrimaryActorTick.bCanEverTick = true;
+}
+
+float ARRNetworkGameState::GetServerWorldTimeSeconds() const
```

Comment:
what is the intension of this function?

it returns client real time or serve rsimulation time?

Reply:
The time stored on the client's player controller (which this function retrieves) is the error corrected Server time if its on the server it just returns the server's time